_layouts/post.html

```diff
@@ -13,6 +13,19 @@
   </aside>
 
   <main id="main-content" class="main-content">
+    {% if page.expiration_date %}
+      <!--
+        Note: `%s` converts the date to "seconds since 1970", based on strftime formatting.
+        This allows us to compare dates to see which is greater.
+      -->
+      {% assign today_date = 'now' | date: '%s' %}
+      {% assign expiration_date = page.expiration_date | date: '%s' %}
+      {% assign expired_content = false %}
```

Comment:
Is this line needed? Looks like it was brought over from the `head.html` expired content logic but doesn't seem to add any value here. Let me know if I'm missing anything!

```suggestion

```

Reply:
Good catch! Removed in [42e3340](https://github.com/uswds/uswds-site/pull/2618/commits/42e33405cbe7083825cfca0dbf952686a64ab815)